english/car/regera.txt

```diff
@@ -0,0 +1 @@
+An hybrid supercar by Koenigsegg. 0-400km/h in 20 holy seconds! and a top speed of 410. Four cars sold before anybody saw the car.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.